config/creds/README.creds

```diff
@@ -0,0 +1,21 @@
+This directory will hold credentials files on the build host.
+
+All JSON files in this directory are ignored via gitignore(5) to reduce the
+risk that credentials accentally get commit.
+
+The following files are currently used:
+
+* gce.json
+
+  A service account JSON file for the Google Cloud Platform. This file can be
+  generated as describe here:
+  https://cloud.google.com/storage/docs/authentication#generating-a-private-key
```

Comment:
I'm good with a very short addition to the README indicating these new features require `...` that we can later cleanup and add more details.